### PR TITLE
adjust directory, more industry

### PR DESCRIPTION
**Pull request by shiningstarpxx**
we put all of the code together in one file, it 's not the real develop structure in real world. 

We adjust the directories to take a small step to real world

### CODE REVIEW

**Review by ychensha — APPROVED** [review]

File: code/app/commodity/svc/db_service.go [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+		DSN:                       "gormuser:gormpassword@tcp(127.0.0.1:3306)/db_commodity?charset=utf8&parseTime=True&loc=Local", // data source name
+		DefaultStringSize:         256,                                                                                            // default size for string fields
+		DisableDatetimePrecision:  true,                                                                                           // disable datetime precision, which not supported before MySQL 5.6
+		DontSupportRenameIndex:    true,                                                                                           // drop & create when rename index, rename index not supported before MySQL 5.7, MariaDB
+		DontSupportRenameColumn:   true,                                                                                           // `change` when rename column, rename column not supported before MySQL 8, MariaDB
+		SkipInitializeWithVersion: false,                                                                                          // auto configure based on currently MySQL version
+	}), &gorm.Config{})
+
+	if err != nil {
+		panic("db init failed")
```
Comment (tirozhang) [inline code/app/commodity/svc/db_service.go:19]: lost err information , adv:panic(err)

File: code/app/commodity/handler/get_product.go [hunk 2]
```diff
@@ -0,0 +1,62 @@
+package handler
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/brahma-adshonor/microservice/code/app/commodity/svc"
+	"net/http"
+
+	"github.com/brahma-adshonor/microservice/code/app/commodity/model"
+	"gorm.io/gorm"
```
Comment (authurguo) [inline code/app/commodity/handler/get_product.go:10]: why not goimports?

File: code/app/commodity/handler/get_product.go [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+)
+
+type ProductRequst struct {
+	Ids []int32 `json:"ids,omitempty"`
```
Comment (authurguo) [inline code/app/commodity/handler/get_product.go:14]: bad style

File: code/app/commodity/handler/get_product.go [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+	All bool    `json:"all,omitempty"`
```
Comment (authurguo) [inline code/app/commodity/handler/get_product.go:15]: usually, tag the boolean value with Is-/Has-/If-

**Review by tirozhang — APPROVED** [review]

**Pull request merged**